### PR TITLE
feat: add the hovering effect to the member cards

### PR DESCRIPTION
**Pull request by xinyue296**
Add a hovering affect to the member cards and updating Mona's profile!

## Description

Add a hovering affect to the member cards.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Review by YHhaoareyou — COMMENTED** [review]
@xinyue296

1. Next time you can use `<a href="#">` instead of `<a href="">` if you don't want anything to happen when clicking the `a` tag.
`href=""` actually navigates you to the same page (just like reloading)

2. Remember to remove comments and unused libraries whenever you finished developing a feature and before submitting a PR

I leave the approve review to @nichnarmada in case he has other comments!

**Review by nichnarmada — CHANGES_REQUESTED** [review]
Please change these!

File: root/src/components/aboutUs/MeetOurTeam/memberList.ts [hunk 1]
```diff
@@ -33,7 +34,7 @@ import twitter from "@app/assets/img/socialmediaicon/twitter.png";
 export interface Members {
   image: string;
   name: string;
-  position: string;
+  positions: Array<{ position: string }>;
```
[the remark this answers is not in the document]
Comment (xinyue296) [reply root/src/components/aboutUs/MeetOurTeam/memberList.ts:37]: I'm not sure what do you mean by array of strings 🥲? Do you mean by this --`positions: Array<{string}>` ?

[the remark this answers is not in the document]
Comment (YHhaoareyou) [reply root/src/components/aboutUs/MeetOurTeam/memberList.ts:37]: @xinyue296 Like this: `["frontend engineer", "SNS manager", "blablabla"]`
If there is only 1 same attribute in each object, you don't actually need to make it an object.
Object is usually used in cases like a `Member` which has more than one attributes, and you use an Array of Objects to manage multiple members.

[the remark this answers is not in the document]
Comment (YHhaoareyou) [reply root/src/components/aboutUs/MeetOurTeam/memberList.ts:37]: So here you can change the type into: `positions: string[]`

[the remark this answers is not in the document]
Comment (xinyue296) [reply root/src/components/aboutUs/MeetOurTeam/memberList.ts]: Thank you for being so helpful, Hao!!! Now it's crystal clear for me 🥳!

**Review by nichnarmada — APPROVED** [review]
There are still some changes I'll want you to do, but we'll do that in the future once storybook is up and running for all projects. Good job!

**Pull request merged**